### PR TITLE
Update app.py -fjrccb

### PR DESCRIPTION
**Pull request by miles-canton**
Update app.py -fjrccb

### CODE REVIEW

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#" title="Author self-approved">miles-canton</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **xq2005** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @xq2005` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by prow-bot-pli** [comment]
Hi @miles-canton. Thanks for your PR.

I'm waiting for a [kubernetes-discovery](https://github.com/orgs/kubernetes-discovery/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/kubernetes-discovery/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: app.py
Comment (tonydongwei) [inline app.py:10]: ok

File: app.py
Comment (tonydongwei) [inline app.py:11]: ok

**Review by tonydongwei — APPROVED** [review]
thank you

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#" title="Author self-approved">miles-canton</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#pullrequestreview-793543840" title="Approved">tonydongwei</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **xq2005** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @xq2005` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by prow-bot-pli** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#issuecomment-955160712" title="Approved">alvin-zhu-github</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#" title="Author self-approved">miles-canton</a>*, *<a href="https://github.com/kubernetes-discovery/pipelines-vote-ui/pull/30#pullrequestreview-793543840" title="Approved">tonydongwei</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=kubernetes-discovery%2Fpipelines-vote-ui).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/kubernetes-discovery/pipelines-vote-ui/blob/master/OWNERS)~~ [alvin-zhu-github]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
